cmd/infra/azure/create.go

```diff
@@ -252,10 +261,23 @@ func createResourceGroup(ctx context.Context, o *CreateInfraOptions, azureCreds
 
 		return *response.ID, *response.Name, existingRGSuccessMsg, nil
 	} else {
+		additionalTags := map[string]*string{}
+
+		if o.ResourceGroupTags != nil {
```

Comment:
Why not just call `ParseAWSTags` here and remove the check above?

If you do that, I'd rename `ParseAWSTags` since it now does it for AWS and Azure. Maybe just `ParseAdditionalTags`?

Reply:
check above is to stop any creation happening if the tags arent formatted properly. Im not reusing it here because the return type of ParseAwsTags is map[string]string and we nedd map[string]*string.

Will rename to avoid confusion.